pkg/samples/samples.go

```diff
@@ -19,55 +18,56 @@ import (
 	"github.com/stripe/stripe-cli/pkg/stripeauth"
 )
 
-var fileEnvRegexp = map[string]*regexp.Regexp{
-	".java": regexp.MustCompile(`String ENV_PATH = "(.*)";`),
-	".js":   regexp.MustCompile(`const envPath = resolve\(__dirname, "(.*)\.env"\);`),
-	".php":  regexp.MustCompile(`\$ENV_PATH = '(.*)';`),
-	".rb":   regexp.MustCompile(`ENV_PATH = '(.*)\.env'\.freeze`),
+type sampleConfig struct {
+	Name            string            `json:"name"`
+	ConfigureDotEnv bool              `json:"configureDotEnv"`
+	PostInstall     map[string]string `json:"postInstall"`
+	Integrations    []integration     `json:"integrations"`
 }
 
-var fileEnvTemplate = map[string]func(string) string{
-	".java": javaPath,
-	".js":   jsPath,
-	".php":  phpPath,
-	".rb":   rbPath,
+func (sc *sampleConfig) hasIntegrations() bool {
+	return len(sc.Integrations) > 1
 }
 
-var languageDisplayNames = map[string]string{
-	"java":   "Java",
-	"node":   "Node",
-	"python": "Python",
-	"php":    "PHP",
-	"ruby":   "Ruby",
+func (sc *sampleConfig) integrationNames() []string {
+	names := []string{}
+	for _, integration := range sc.Integrations {
+		names = append(names, integration.Name)
+	}
+
+	return names
 }
 
-var displayNameLanguages = reverseStringMap(languageDisplayNames)
+func (sc *sampleConfig) integrationServers(name string) []string {
+	for _, integration := range sc.Integrations {
+		if integration.Name == name {
+			return integration.Servers
+		}
+	}
 
-func javaPath(path string) string {
-	return fmt.Sprintf(`String ENV_PATH = "%s/";`, path)
+	return []string{}
 }
 
-func jsPath(path string) string {
-	return fmt.Sprintf(`const envPath = resolve(__dirname, "%s/.env");`, path)
+type integration struct {
```

Comment:
Would you mind if we renamed this to `sampleConfigIntegration` so that it's really easy to see that the two structs are related? (And both used for configuration.)

Reply:
Done!